doc/COMPILE.md

```diff
@@ -712,6 +712,32 @@ For reference, you can look into the
 and the
 [CMake in OpenWRT](https://github.com/openwrt/openwrt/blob/master/include/cmake.mk).
 
+### Resolving Missing \*.so Library Error
```

Comment:
What you write here is only about run-time problems, it is unrelated to compiling (and the recommended solution of how to deal with .so is already in this document a bit above (about RPATH).

Maybe best if you put a condensed version of this directly to contributing-clion.md because it is not enough for its own tutorial and I would actually not recommend to do it this way, except if the IDE does not support to run scripts/run_dev_env

Reply:
Well, you said it yourself I should put in into compile.md. Sure, it makes more sense to have in contributing-clion.md.

Reply:
Sorry, for the confusion. I pointed you to COMPILE.md because there is already a description of the solution to your problem. The "library" not found is a topic that spreads from compiling, installing to run-time. Your solution is run-time only, though.